auditlog/migrations/14.0.1.1.0/pre-migration.py

```diff
@@ -0,0 +1,73 @@
+# © 2018 Pieter Paulussen <pieter_paulussen@me.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+import logging
+
+
+def migrate(cr, version):
+    if not version:
+        return
+    logger = logging.getLogger(__name__)
+    logger.info(
+        "Creating columns: auditlog_log (model_name, model_model) "
+        "and auditlog_log_line (field_name, field_description)."
+    )
+    cr.execute(
+        """
+    ALTER TABLE auditlog_log
```

Comment:
The alignment of those lines seems odd

Reply:
Same as https://github.com/OCA/server-tools/blob/8.0/auditlog/migrations/8.0.1.4.0/pre-migration.py. I don't think there is a leading convention on the formatting of multiline SQL statements so I would prefer to leave it like this.

Reply:
The only change was applied to the position of the quotes, by python-black in the prehook, and we can't derive from that.